reals.tex

```diff
@@ -3072,7 +3072,7 @@ \section{The surreal numbers}
 
 \begin{ex} \label{ex:RD-extended-reals}
   %
-  Suppose we remove the boundedness condition in \cref{defn:dedekind-reals}.
+  Suppose we remove the \emph{inhabited} condition in \cref{defn:dedekind-reals}.
```

Comment:
The phrasing is not the best, because it's not the "inhabited condition" that is removed, but "the condition that asserts that the upper and lower sets are inhabited" that is removed.

Reply:
> The phrasing is not the best, because it's not the "inhabited condition" that is removed, but "the condition that asserts that the upper and lower sets are inhabited" that is removed.

By analogy to Exercise 11.4 I suppose another option is "the inhabitedness condition" or even "both inhabitedness conditions".

I guess I don't object to "the condition that asserts that the upper and lower sets are inhabited" although I guess we usually try to be more concise? Something like "the condition that the upper and lower cuts are inhabited" is slightly shorter.

I'm sold on at least making it "inhabitedness" (without italics) instead of "inhabited" but from the possibilities above I'm not sure I have an opinion.

Reply:
"inhabitedness" sounds bad to my ear